src/logger/MetricsContext.ts

```diff
@@ -118,12 +118,12 @@ export class MetricsContext {
     // This ensures the latest dimension key-value is used as a target member on the root EMF node.
     // This operation is O(n^2), but acceptable given sets are capped at 10 dimensions
     const incomingDimensionSetKeys = Object.keys(incomingDimensionSet);
-    this.dimensions = this.dimensions.filter(existingDimensionSet => {
+    this.dimensions = this.dimensions.filter((existingDimensionSet) => {
```

Comment:
@markkuhn is this necessary change?

Reply:
This change was done by the linter to comply with our styling config